lib/src/css_parser.dart

```diff
@@ -41,6 +41,26 @@ Style declarationsToStyle(Map<String, List<css.Expression>> declarations) {
       case 'text-align':
         style.textAlign = ExpressionMapping.expressionToTextAlign(value.first);
         break;
+      case 'text-decoration':
```

Comment:
This code confuses me. Partly because it modifies list inline, partly because of the generic variable names. What is `exp1`, `exp2` and `exp3`? Do I understand that these are actually the line, color and style? It feels like these cold be written as easier to read one liners, such as
```dart
style.textDecorationColor = ExpressionMapping.expressionToColor(value.firstWhere((element) => element is css.HexColorTerm || element is css.FunctionTerm, orElse: null));
```

Reply:
Yea I should have made those more clear... exp1 is the decoration type, exp2 is the color, and exp3 is the style. 

I tried using that one liner first but it didn't work, it still kept the `css.Expression` elements inside the list it created. That's why I had to use the `whereType<>()` function.

Reply:
Fixed.